aleph-client/src/session.rs

```diff
@@ -58,6 +58,27 @@ pub fn change_members(
     );
     send_xt(sudo_connection, xt, Some("sudo_unchecked_weight"), status);
 }
+pub fn change_reserved_members(
+    sudo_connection: &RootConnection,
+    new_members: Vec<AccountId>,
+    status: XtStatus,
+) {
+    info!(target: "aleph-client", "New members {:#?}", new_members);
+    let call = compose_call!(
+        sudo_connection.as_connection().metadata,
+        "Elections",
+        "change_reserved_members",
+        new_members
+    );
+    let xt = compose_extrinsic!(
+        sudo_connection.as_connection(),
+        "Sudo",
+        "sudo_unchecked_weight",
+        call,
+        0_u64
+    );
+    send_xt(sudo_connection, xt, Some("sudo_unchecked_weight"), status);
```

Comment:
Uh, I would rename this xt to something related to changing reserved members

Reply:
isn't the whole function called `change_reserved_members`?

Reply:
yeah, but in logs you will see only `"Transaction sudo_unchecked_weight was included in block with hash {}."`, which I think is less useful that  `"Transaction change_reserved_members was included in block with hash {}."`